app/src/main/java/com/expidev/gcmapp/MeasurementsActivity.java

```diff
@@ -110,6 +110,11 @@ private void restartMeasurementLoader(String ministryId, String mcc, String peri
 
     private void drawLayout(Ministry selectedMinistry, String mcc, List<Measurement> measurements)
```

Comment:
not something that needs to be done for this PR, but you should look into how to utilize a ListView (or RecyclerView since Lollipop/support library 21) with a custom item layout to generate the views for all the measurements.

It has the benefit of reusing views as they scroll off the screen (less memory usage) and allowing the layout/UI to be customized in an XML layout.